whatsnew/createnewandnoteworthy.py

```diff
@@ -82,13 +82,14 @@ def quote(x):
     
     cms = map(nametuple, comps)    
     #print name + "->" + str(cms)
+    new_file_contents = '= '+ name +' What\'s New\n:page-layout: whatsnew\n:page-component_id: '+comps+'\n:page-component_version: '+jbide_fixversion+'\n:page-product_id: jbt_core\n:page-product_version: '+jbide_fixversion
```

Comment:
Ah, I see what you mean, maybe having a placeholder in the file contents instead?